test/resources/issue-101.js

```diff
@@ -0,0 +1,63 @@
+'use strict'
+
+function names(arg) {
+  const flag = arg
+  const addPrefix = (value) => (flag ? `"${value}"` : `"my_prefix.${value}"`)
+  const result = `
+      ${addPrefix('NAME_0')}
+      ${addPrefix('NAME_1')}
+      ${addPrefix('NAME_2')}
+      ${addPrefix('NAME_3')}
+    `
+  return result
+}
+
+function namesBlock(arg) {
+  const flag = arg
+  const addPrefix = (value) => {
+    return flag ? `"${value}"` : `"my_prefix.${value}"`
+  }
+  const result = `
+      ${addPrefix('NAME_0')}
+      ${addPrefix('NAME_1')}
+      ${addPrefix('NAME_2')}
+      ${addPrefix('NAME_3')}
+    `
+  return result
+}
+
+function namesNested(arg) {
+  const flag = arg
+  const addSufix = (value) => `(${!value}_suffix`
+  const addPrefix = (value) => (flag ? `"${value}"` : `"my_prefix.${addSufix(value)}"`)
+  const result = `
+      ${addPrefix('NAME_0')}
+      ${addPrefix('NAME_1')}
+      ${addPrefix('NAME_2')}
+      ${addPrefix('NAME_3')}
+    `
+  return result
+}
+
```

Comment:
Could you try to do also these tests:

```suggestion

function namesDoubleResult(arg) {
  const flag = arg
  const addPrefix = (value) => (flag ? `"${value}"` : `"my_prefix.${value}"`)
  const resultFake = `
      ${addPrefix('NAME_0')}
      ${addPrefix('NAME_1')}
      ${addPrefix('NAME_2')}
      ${addPrefix('NAME_3')}
    `
  const result = `
      ${addPrefix('NAME_0')}
      ${addPrefix('NAME_1')}
      ${addPrefix('NAME_2')}
      ${addPrefix('NAME_3')}
    `
  return result
}

function namesNoFlag() {
  const addPrefix = (value) => `"my_prefix.${value}"`
  const result = `
      ${addPrefix('NAME_0')}
      ${addPrefix('NAME_1')}
      ${addPrefix('NAME_2')}
      ${addPrefix('NAME_3')}
    `
  return result
}

function namesNoFlagDoubleResult() {
  const addPrefix = (value) => `"my_prefix.${value}"`
  const resultFake = `
      ${addPrefix('NAME_0')}
      ${addPrefix('NAME_1')}
      ${addPrefix('NAME_2')}
      ${addPrefix('NAME_3')}
    `
  const result = `
      ${addPrefix('NAME_0')}
      ${addPrefix('NAME_1')}
      ${addPrefix('NAME_2')}
      ${addPrefix('NAME_3')}
    `
  return result
}
```